packages/venia-ui/lib/tokens.css

```diff
@@ -85,8 +85,13 @@
 
     /* dimensions */
     --venia-global-maxWidth: 1440px;
-    --venia-global-header-minHeight: 80px;
-    --venia-mobile-global-header-minHeight: 56px;
+    --venia-global-header-minHeight: 5rem;
+}
+
+@media (max-width: 960px) {
```

Comment:
So we want an additional one at 641px I believe.

Reply:
I don't understand why we might need another one at 640. Wouldnt `960` cover for `640` as well? I didn't want to introduce new UI changes as part of this PR. I only wanted to move the existing hardcoded values to tokens.

Reply:
Ok I think I was confused - I thought the store switcher would affect the height, since it hides below 640px. So my concern was how we would handle the scrolling for widths between 640 and 960.

What I did not know was that we auto-hide the switcher when you scroll down, and only the header is sticky. Because of that, only the two possible header sizes matter, so you've covered those cases :)